nilearn/_utils/data_gen.py

```diff
@@ -677,7 +677,7 @@ def generate_group_sparse_gaussian_graphs(n_subjects=5,
     topology = topology > 0
     assert (np.all(topology == topology.T))
     logger.log(
-        "Sparsity: {0:f}".format(1. * topology.sum() / (topology.shape[0]**2)),
+        f"Sparsity: {1.0 * topology.sum() / topology.shape[0] ** 2:f}",
```

Comment:
```suggestion
        f"Sparsity: {1.0 * topology.sum() / topology.shape[0] ** 2 :f}",
``` would be more readable IMHO

Reply:
let's see what black has to say about this